### DISCUSSION TITLE
Zone repeat x1 but twice on carpet in this selection.

### DISCUSSION THREAD

**Discussion by gbdmpl** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#324] [General]
Hi,

i have a zone which i want to clean with repeat 1x, but i have carpet in this selection which i want repeat x2. if this is possible with "one click"?

**Comment by PiotrMachowski** [comment]
Nope, currently card supports just one global repeats counter

**Reply by gbdmpl** [reply]
ok, thanks

good job/świetna robota
